src/azure-cli-core/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.0.18 (2017-10-06)
```

Comment:
this should say `unreleased` as per https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log